lib/sorcery/model/submodules/jwt.rb

```diff
@@ -33,7 +33,7 @@ def self.included(base)
 
         module ClassMethods
           def issue_token(payload)
-            final_payload = payload.merge(exp: Time.now.to_i + @sorcery_config.session_expiry)
+            final_payload = payload.merge(exp: Time.now.to_i + @sorcery_config.session_expiry, iat: Time.now.to_i)
```

Comment:
@dxin93 What does IAT stand for? And why do we need to bake it into the payload of the token? Does Chase need that on the FE?

Reply:
Happy to keep `iat` as a var name, but let's add a comment please with some context. Also is there a test suite that we could enhance, or would we have to write a lot of boiler plate code to get a first test built?

Reply:
@mklocker `iat` is the JWT standard for "Issued At Time" https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.6

Chase thought we might need it if the client were to re-issue tokens themselves, but I wasn't quite clear on why. I just followed up and it looks like we don't actually need this unless it's explicitly requested from us.